articles/introduction-to-airflow/index.md

```diff
@@ -0,0 +1,110 @@
+
+![dags](/engineering-education/introduction-to-airflow/dag.png)
+
+#### Operators
+
+According to the documentation, an operator describes a single task in a workflow, and are usually (but not always) atomic, meaning they can stand on their own and don’t need to share resources with any other operators. In other words, an operator determines what a task does in the DAG. Some operators that Airflow provides are:
+
+[BashOperator](https://airflow.apache.org/docs/stable/_api/airflow/operators/bash_operator/index.html#airflow.operators.bash_operator.BashOperator) - This operator can be used to execute bash commands in Python.
+
+[PythonOperator](https://airflow.apache.org/docs/stable/_api/airflow/operators/python_operator/index.html#airflow.operators.python_operator.PythonOperator) - This operator can call functions written in Python.
+
+[EmailOperator](https://airflow.apache.org/docs/stable/_api/airflow/operators/email_operator/index.html#airflow.operators.email_operator.EmailOperator) - This operator can be used to send emails to one or more recipients.
+
+[SimpleHttpOperator](https://airflow.apache.org/docs/stable/_api/airflow/operators/http_operator/index.html#airflow.operators.http_operator.SimpleHttpOperator) - This operator is used to send HTTP requests.
+
+### Installing Airflow
+
+Since Airflow is written completely in Python, we can use [pip](https://pip.pypa.io/en/stable/) (The package manager for Python). You can install Airflow on your local system by running the following command on the terminal:
+
+```bash
+pip3 install apache-airflow
+```
+
+Airflow configuration files and database info will be stored in the environment variable called AIRFLOW_HOME. By default, Airflow will create the directory ~/airflow/, but you can change the default path to a directory of your choice. To change the airflow home path, enter the following command on the terminal:
+
+```bash
+export AIRFLOW_HOME=<the path to the directory of your choice>
+```
+
+Example:
+```bash
+export AIRFLOW_HOME=~/adith/airflow
+```
+
+Airflow uses a database in the back-end to store metadata and other workflow-related information. By default, [SQLite](https://www.sqlite.org/index.html), a lightweight, and beginner-friendly database is used by Airflow. You can change the default settings to a database of your choice by following [this documentation](https://airflow.apache.org/docs/stable/howto/initialize-database.html). Since SQLite is a beginner-friendly database, we are going to stick with that.
```

Comment:
A beginner might not understand what metadata means. You could include a couple of examples